docs/sphinx/users/micromanager/index.txt

```diff
@@ -0,0 +1,39 @@
+Micro-Manager
+=============
+
+- Micromanager offers the functionality for saving the acquired images in tiff/ome-tiff format. 
+- Based on the mode of saving and the configuration settings, the acquired image can be saved 
+  with/without a companion file (metadata.txt).
+
+.. figure:: /images/micromanager_saving.png
+    :width: 180px
+    :align: center
+    :alt: Saving options in Micro-Manager
+
+
++---------------+----------+--------------+--------------+--------------------+
+|Saving         |          |              |              |                    |
+|Options within |          |              |              |                    |
+|Micro-Manager  |          |Companion     |              |                    | 
+|(Default       |          |File          |Bio-Formats   |                    |
+|Configuration) |Format    |(metadata.txt)|Reading       | Reader Used        |
```

Comment:
This reads as if the saving options (image stack vs separate images) would be different with a non-default configuration.